FreeSurfer-DockerFiles/Dockerfile.exec-centos5.freesurferbuild-centos4

```diff
@@ -48,4 +48,4 @@ ENV PATH=${PATH}:${FREESURFER_HOME}/bin
 ADD ./freesurfer-command-line-script.sh /bin
 
 #Add write permission
-RUN chmod +w /usr/local/src -R
+RUN ls -l /usr/local/src; chmod +w -R;
```

Comment:
should be `RUN ls -l /usr/local/src; chmod +w -R /usr/local/src`. You're missing the path in the `chown` command.